app/views/foreman/smart_proxies/_content_sync.html.erb

```diff
@@ -14,6 +14,13 @@
     </a>
   </div>
 
+  <div ng-hide="syncStatus.unsyncable_content_types.length == 0">
```

Comment:
The spacing of the divs on this page really buries this message. Some ways to make this "pop" more (which I would say is the goal of the PR) would be to:

- increase the spacing around these divs
- perhaps move this new div to the top instead of in between the sync time and status divs **or** put it right next to the Synchronize button?

Reply:
@jturel what do you think of this?

![Screenshot from 2021-11-11 10-27-43](https://user-images.githubusercontent.com/14796566/141324575-7462647e-3da2-4e0f-be6a-ef59ce3a93ed.png)

Reply:
Nice!